### DISCUSSION TITLE
I can't install OpenVR-SpaceCalibrator

### DISCUSSION THREAD

**Discussion by astrokim-dk** [community/community#43608] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

hi i don't know who to contact about this.
but I can't install OpenVR-SpaceCalibrator
can you help me what could be wrong

Error opening file for writing:

Failed to initialize OpenVR Installation path could not be located (110)Unable to read VR Path Registry from QUsers\astro\AppData\Local\openvr\openvrpaths.vrpath\driv ers\01spacecalibrator\driver.vrdrivermanifest

Click Abort to stop the installation, Retry to try again, or Ignore to skip this file.

**Comment by jfullstackdev** [comment]
you can raise an Issue there and ask the same question 

https://github.com/pushrax/OpenVR-SpaceCalibrator
